plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/__init__.py

```diff
@@ -15,8 +15,8 @@
 @dataclass
 class RayLauncherConf:
     _target_: str = "hydra_plugins.hydra_ray_launcher.ray_launcher.RayLauncher"
-    ray_init_cfg: Dict[str, Any] = field(default_factory=dict)
-    ray_remote_cfg: Dict[str, Any] = field(default_factory=dict)
+    ray_init: Dict[str, Any] = field(default_factory=lambda: {"address": None})
```

Comment:
to make this easier to read, add a trailing comma:
```
ray_init: Dict[str, Any] = field(default_factory=lambda: {"address": None,})
```
and let black format it.

Reply:
nice!